synapse/handlers/message.py

```diff
@@ -888,7 +890,7 @@ async def persist_and_notify_client_event(
 
         This should only be run on master.
```

Comment:
Not sure how our terminology is changing, but I think this docstring should be updated?